crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -174,8 +174,15 @@ impl Query {
         DryRunResult::try_from(res).extend()
     }
 
-    async fn owner(&self, address: SuiAddress) -> Option<Owner> {
-        Some(Owner { address })
+    async fn owner(&self, ctx: &Context<'_>, address: SuiAddress) -> Result<Owner> {
```

Comment:
The draft schema indicates this should be optional, so just wanted to mention to double check.